companion/src/firmwares/opentx/opentxinterface.cpp

```diff
@@ -1459,6 +1469,16 @@ void registerOpenTxFirmwares()
   addOpenTxRfOptions(firmware, NONE);
   firmware->addOption("bindkey", Firmware::tr("Allow bind using bind key"));
 
+  /* iFlight Commando8 board */
+  firmware = new OpenTxFirmware(FIRMWAREID("commando8"), QCoreApplication::translate("Firmware", "iFlight Commando8"), BOARD_IFLIGHT_COMMANDO8);
+  addOpenTxCommonOptions(firmware);
+  firmware->addOption("noheli", Firmware::tr("Disable HELI menu and cyclic mix support"));
+  firmware->addOption("nogvars", Firmware::tr("Disable Global variables"));
+  firmware->addOption("lua", Firmware::tr("Enable Lua custom scripts screen"));
```

Comment:
Please be aware that the compilation options are not functional for now. The 3 options listed here are also on the list of options that will probably be removed soon (selected only at run-time).